.github/workflows/deployment.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         shell: bash
         run: |
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
-          EPHEMERAL_STORAGE=True bash openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
+          EPHEMERAL_STORAGE=False bash openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
```

Comment:
Why this change?

Reply:
Yeah, I was going to reach out to you about this and the changes to PVC sizes. In order to conserve resources we don't allocate 'real' PVCs for dev deployments and instead rely on transient storage of some sort (I think). In order to test this PR, we needed large PVCs so that I could throw large quantities of data into the PR patroni instance and create the materialized views. 

I'll roll back this change and revert the changes to openshift/templates/patroni.yaml and openshift/templates/patroni_prerequisite.yaml.

Reply:
All good makes sense, this comment was mostly me showing @brettedw the code review process :)